homeassistant/components/withings/__init__.py

```diff
@@ -39,68 +26,11 @@
     get_data_manager_by_webhook_id,
     json_message_response,
 )
-from .const import CONF_USE_WEBHOOK, CONFIG, LOGGER
+from .const import CONF_USE_WEBHOOK, LOGGER
 
 DOMAIN = const.DOMAIN
 PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
 
-CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.All(
```

Comment:
May be people should be warned they still have this stuff in their `configuration.yaml`?

Reply:
Yes they should, I'm one of those people 😉